src/Resources/app/storefront/src/js/filter-property-select.plugin.js

```diff
@@ -0,0 +1,42 @@
+import FilterPropertySelectPlugin from 'src/plugin/listing/filter-property-select.plugin';
```

Comment:
The size filter and color/vendor image is properly disabled:

![image](https://user-images.githubusercontent.com/20555112/106873990-58dedd80-66d5-11eb-8c99-5b77f1ec0ac2.png)

![image](https://user-images.githubusercontent.com/20555112/106874242-a5c2b400-66d5-11eb-88ce-417c7a889b1f.png)


But it doesn't seem to be the case for content or textile, so I am still able to get no results:

![image](https://user-images.githubusercontent.com/20555112/106874125-7f9d1400-66d5-11eb-8774-19fc143b1722.png)

Please ensure this properly works.

Reply:
@TheKeymaster 
I had tested it and it was working. These all are the same filter type so we are not doing anything different for these. Can you verify the findologic API response. It happens that at times it sends all the filters back instead of only selected ones.

Reply:
I will have another look at it as well.

Reply:
As discussed, will be fixed in SW-578.